test/hardhat/smart-account/Nexus.Module.K1Validator.specs.ts

```diff
@@ -236,45 +236,47 @@ describe("K1Validator module tests", () => {
       expect(isValid).to.equal(1);
     });
 
-    it("should sign with eth_sign", async () => {
-      const isModuleInstalled = await deployedNexus.isModuleInstalled(
-        ModuleType.Validation,
-        k1ModuleAddress,
-        ethers.hexlify("0x"),
-      );
+    // Review: below test started failing.
```

Comment:
@filmakarov thoughts?

Reply:
oh right! buildPackedUserOp returns signature = "0x"

Reply:
I think it didn't fail before cause SignatureCheckerLib doesn't revert for above input?

Reply:
just try to properly sign it or rm as `eth_sign` is outdated

Reply:
maybe should cover some negative tests though

Reply:
yeah, that would be great

Reply:
checked foundry test cases covers